packages/synapse-react-client/src/components/file/upload/BasicFileHandleUpload.tsx

```diff
@@ -0,0 +1,73 @@
+import MultiFileUploadProgress from '@/components/file/upload/MultiFileUploadProgress'
+import UploadFilePanel from '@/components/file/upload/UploadFilePanel'
+import {
+  UploaderState,
+  useUploadFiles,
+} from '@/utils/hooks/useUploadFileEntity/useUploadFiles'
+import { noop } from 'lodash-es'
+import { ForwardedRef, forwardRef, useEffect, useImperativeHandle } from 'react'
+
+export type BasicFileHandleUploadProps = {
+  allowMultipleUpload: boolean
+  /** Callback that is invoked when the state of the uploader changes */
+  onStateChange?: (state: UploaderState) => void
+  /** Callback that is invoked when component is ready to upload */
+  onUploadReady?: () => void
+}
+
+export type FileUploadHandle = {
+  /** Programmatically add files to the upload (e.g. on drag & drop) */
+  handleUploads: (fileList: ArrayLike<File>) => void
+}
+
+/**
+ * BasicFileHandleUpload is a component that supports uploading a file to the default Synapse S3 bucket and creating a
+ * file handle, which can only be accessed the system itself and the user uploading the file.
+ */
+export const BasicFileHandleUpload = forwardRef(function FileHandleUpload(
```

Comment:
New component that handles uploading one more more files (including the UI for drag and drop), without any of the logic for creating/updating entities.

Not yet used (coming in a later change) so this may be subject to a few minor changes.